tachyon/crypto/sumcheck/BUILD.bazel

```
@@ -20,6 +20,14 @@ tachyon_cc_library(
     ],
 )
 
+tachyon_cc_library(
+    name = "sumcheck_verifier_msg",
+    hdrs = ["sumcheck_verifier_msg.h"],
+    deps = [
+        "//tachyon/base/buffer:copyable",
+    ],
```

It's not a mandatory, but could you inline it?

```suggestion
    deps = ["//tachyon/base/buffer:copyable"],
```